domains/Links/Tests/Feature/LinksStoreTest.php

```diff
@@ -40,7 +40,7 @@ public function it_stores_resources(): void
         Storage::fake('local');
 
         $payload = [
-            'link' => $this->faker->url,
+            'link' => 'https://www.google.com',
```

Comment:
Thanks for this, but you can revert this. I'll fix the rule that's breaking this, in the meanwhile.

Reply:
done